examples/fluids/navierstokes.h

```diff
@@ -211,6 +211,11 @@ struct SetupContext_ {
   CeedScalar dc_axis[3];
   CeedScalar wind[3];
   CeedScalar time;
+  CeedScalar mid_point;
+  CeedScalar P_high;
+  CeedScalar rho_high;
+  CeedScalar P_low;
+  CeedScalar rho_low;
```

Comment:
Ah, HIP is failing because these are not added to SetupContext in `qfunctions/advection.c`, `qfunctions/advection2d.c`, and `qfunctions/densitycurrent.c`. You would also need to add this Context to `qfunctions/shocktube.h` to make HIP happy.

Reply:
Is there any action for me here? I can add these to the relevant files if needed.

Reply:
Yeah, the GPU tests won't pass if we don't add them. Do you mind making this change?

Reply:
Done!

Reply:
Thanks! Could you add SetupContext to shocktube.h as well?